src/components/Note/Note.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+import { Note, TextLink } from '@contentful/forma-36-react-components';
+
+export interface INoteProps {
+  type: 'primary' | 'positive' | 'negative' | 'warning';
+  error: Error;
+  resetErrorBoundary: () => void;
+}
+
+export function IxNote({
+  error,
+  type,
+  resetErrorBoundary,
+}: INoteProps): ReactElement {
+  const [message, _link, ...rest] = error.message.split('$');
+  const [link, title] = _link.split('|');
+  const linkTitle = title?.length ? title : link;
```

Comment:
This component parses the `error.message` value to pull out a link href
and title.

The error messages are split on `$` and then again on `|`.
The `$` is used to parse the message's "link" URL. The`|` is used to
parse the message's "link" title. If the `message` string has no `|`
then the `$` is used to parse the message's "link" URL and title.


`Return to $https://app.contentful.com/deeplink?link=apps|App Config Page$ to enter a valid API key`

Will be parsed as:

`Return to App Config Page to enter a valid
API key`.

Reply:
Thanks for the thorough explanation 👍